install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -376,7 +376,7 @@ spec:
           {{- toYaml .Values.nodeinit.resources | trim | nindent 10 }}
 {{- end }}
       restartPolicy: Always
-{{- if or (and (eq .Release.Namespace "kube-system") (gt .Capabilities.KubeVersion.Minor "10")) (ge .Capabilities.KubeVersion.Minor "17") (gt .Capabilities.KubeVersion.Major "1")}}
+{{- if and (or (and (eq .Release.Namespace "kube-system") (gt .Capabilities.KubeVersion.Minor "10")) (ge .Capabilities.KubeVersion.Minor "17") (gt .Capabilities.KubeVersion.Major "1")) (not .Values.gke.enabled) }}
```

Comment:
What happens once GKE enables it? Also, this means that none of the GKE versions accepts the `priorityClassName`, did we verify this?

Reply:
You are right, I will recrete the flag without double negation.